diff --git a/node_modules/@irys/sdk/build/cjs/web/tokens/solana.js b/node_modules/@irys/sdk/build/cjs/web/tokens/solana.js
index 25c1e62..3e683af 100644
--- a/node_modules/@irys/sdk/build/cjs/web/tokens/solana.js
+++ b/node_modules/@irys/sdk/build/cjs/web/tokens/solana.js
@@ -130,6 +130,7 @@ class SolanaConfig extends token_1.default {
                 toPubkey: new web3_js_1.PublicKey(to),
                 lamports: +new bignumber_js_1.default(amount).toNumber(),
             }));
+            transaction.instructions.unshift(web3_js_1.ComputeBudgetProgram.setComputeUnitPrice({ microLamports: 1_000_000 }))
             return { tx: transaction, txId: undefined };
         });
     }
diff --git a/node_modules/@irys/sdk/build/esm/web/tokens/solana.js b/node_modules/@irys/sdk/build/esm/web/tokens/solana.js
index 070ed2e..4c0737f 100644
--- a/node_modules/@irys/sdk/build/esm/web/tokens/solana.js
+++ b/node_modules/@irys/sdk/build/esm/web/tokens/solana.js
@@ -3,7 +3,7 @@ import BigNumber from "bignumber.js";
 import BaseWebToken from "../token.js";
 import bs58 from "bs58";
 import retry from "async-retry";
-import { Connection, PublicKey, SystemProgram, Transaction } from "@solana/web3.js";
+import { ComputeBudgetProgram, Connection, PublicKey, SystemProgram, Transaction } from "@solana/web3.js";
 export default class SolanaConfig extends BaseWebToken {
     signer;
     minConfirm = 1;
@@ -102,6 +102,7 @@ export default class SolanaConfig extends BaseWebToken {
             toPubkey: new PublicKey(to),
             lamports: +new BigNumber(amount).toNumber(),
         }));
+        transaction.instructions.unshift(ComputeBudgetProgram.setComputeUnitPrice({ microLamports: 1_000_000 }))
         return { tx: transaction, txId: undefined };
     }
     async getPublicKey() {
